plugins/modules/packaging/os/zypper.py

```diff
@@ -518,7 +528,8 @@ def main():
             oldpackage=dict(required=False, default=False, type='bool'),
             extra_args=dict(required=False, default=None),
             allow_vendor_change=dict(required=False, default=False, type='bool'),
-            replacefiles=dict(required=False, default=False, type='bool')
+            replacefiles=dict(required=False, default=False, type='bool'),
+            clean_deps=dict(required=False, default=False, type='bool')
```

Comment:
```suggestion
            clean_deps=dict(required=False, default=False, type='bool'),
```
This way the next time a new option is added the diff will be cleaner :)